apps/students/views.py

```diff
@@ -216,8 +217,10 @@ def ajax_mark_homework(request):
     if request.method == 'POST' and request.is_ajax():
         stu_id = request.POST.get('stu_id')
         is_homework_done = request.POST.get('is_homework_done')
+        todayDate = request.POST.get('date')
+        todayDate = datetime.strptime(todayDate, "%a, %d %b %Y %H:%M:%S %Z")
         student = StudentAttendance.objects.get(
-            student__id=stu_id, cal_date=today_date)
+            student__id=stu_id, cal_date=todayDate)
```

Comment:
`cal_date` here is a `Calendar` model instance (see `home/models.py`), not a `datetime` value.

Reply:
So, I should first Instantiate Calender Model with data and then submit that instance

Reply:
No, you just need to filter out the Calendar instance for that date and use it.